pmd-java/src/test/resources/net/sourceforge/pmd/lang/java/metrics/impl/xml/AtfdTest.xml

```diff
@@ -129,7 +129,7 @@ public class StatementAndBraceFinder extends JavaParserVisitorAdapter {
         <description>Full example</description> <!-- TODO issues w/ that-->
         <expected-problems>7</expected-problems>
         <expected-messages>
-            <message>'StatementAndBraceFinder' has value 10 highest 6.</message>
+            <message>'StatementAndBraceFinder' has value 36 highest 18.</message>
```

Comment:
I see, now the sum actually matches all the methods.... 36=6+4+0+4+4+18 - and the biggest contributor is the method with 18